src/main/java/com/egen/model/Order.java

```diff
@@ -1,17 +1,194 @@
 package com.egen.model;
 
+import javax.persistence.*;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@NamedQueries({
+		@NamedQuery(name = "Order.findAll", query = "SELECT order FROM Order order JOIN fetch order.shippingAddress JOIN FETCH order.itemsList"),
+		@NamedQuery(name = "Order.findById", query = "SELECT *  from Order where orderId = :orderId"),
+		@NamedQuery(name = "Order.findWithinInterval", query = "SELECT order FROM Order order JOIN fetch order.shippingAddress JOIN fetch order.itemsList WHERE order.createdAt > :startTime AND order.createdAt < :endTime"),
+		@NamedQuery(name = "Order.findTop10OrdersWithHighestDollarAmountInZip", query = "")
+
+})
 public class Order {
```

Comment:
order is a key word in most dbms. Use @table annotation to change the table name